README.md

```diff
@@ -1,7 +1,12 @@
 # TRENZ.Lib.RazorMail
 ## Templated transactional e-mail using Razor
 
-This is a simple library you can use to write e-mail templates in [Razor syntax](https://learn.microsoft.com/en-us/aspnet/core/mvc/views/razor). That means you write raw HTML, but elevated with C# — you get `@foreach`, `@switch`, and so on, _and_ you get a strongly-typed model for custom data.
+This is a simple library you can use to write e-mail templates in [Razor syntax](https://learn.microsoft.com/en-us/aspnet/core/mvc/views/razor). That means you write raw HTML,
+but elevated with C# — you get `@foreach`, `@switch`, and so on, _and_ you get a strongly-typed model for custom data.
+
+In NuGet, reference either the `TRENZ.Lib.RazorMail.SystemNet` or the `TRENZ.Lib.RazorMail.MailKit` package, depending
+on which `MailSender` backend you prefer. [MailKit](https://github.com/jstedfast/MailKit) is more modern and powerful,
+but `System.Net.Mail` comes built into .NET. There is no need to reference `TRENZ.Lib.RazorMail` directly.
 
 A simple template looks like so:
```

Comment:
Can we add installation instructions here?

```suggestion
Install via `dotnet`:

`dotnet add package TRENZ.Lib.RazorMail.SystemNet` or `dotnet add package TRENZ.Lib.RazorMail.MailKit`

A simple template looks like so:
```

Or sth like that

Reply:
yeah. I think the README could use some more love, perhaps with more examples, but I've added the above.